exdb/views.py

```diff
@@ -165,3 +185,25 @@ def get_context_data(self, **kwargs):
         context = super(ViewExperienceView, self).get_context_data()
         context['experience'] = get_object_or_404(Experience, pk=self.kwargs['pk'])
         return context
+
+
+class EditExperienceView(UpdateView):
+    access_level = 'basic'
+    template_name = 'exdb/edit_experience.html'
+    form_class = ExperienceSubmitForm
+    model = Experience
+
+    def get_success_url(self):
+        return reverse('ra_home')
+
+    def get_context_data(self, **kwargs):
+        context = super(EditExperienceView, self).get_context_data()
+        experience = get_object_or_404(Experience, pk=self.kwargs['pk'])
```

Comment:
You should define `get_queryset` for security purposes. Make it restrict the people who can edit the Experience to planners and keep them from editing it after the start date we'll ask Travis about exactly when they should no longer be allowed to edit it.
Add a `prefetch_related` to get the `ExperienceComment`'s.
Delete the entire `get_context_data` function because it isn't necessary. You should assign `context_object_name = 'experience'` and use that in the template to get the status and the comments.